.github/workflows/fronts.yml

```diff
@@ -0,0 +1,40 @@
+name: DCR Fronts–Ophan Components
+on:
+  push:
+    branches: [main]
+
+  schedule:
+    # Every work day of the week at 08:08
+    - cron: "8 8 * * MON-FRI"
```

Comment:
So this is on every push to main and once a day on the work week?

Reply:
Whoops, good point, if we’re doing once a week we can remove the pushes!